osidb/api_views.py

```diff
@@ -345,6 +354,20 @@ def create(self, request, *args, **kwargs):
         return response
 
 
+@include_meta_attr_extend_schema_view
+@include_exclude_fields_extend_schema_view
+@extend_schema_view(
+    create=extend_schema(
+        request=FlawAcknowledgmentPostSerializer,
+    ),
+)
+class FlawAcknowledgmentView(ModelViewSet):
```

Comment:
Destroy method should also return code 200 instead of 204 because we inject some default info about OSIDB like:

```
  "dt": "2023-07-03T07:22:10.661Z",
  "env": "string",
  "revision": "string",
  "version": "string"
  ```

and 204 response by standard won't include any response body.

Reply:
Fixed in commit 3c2d2dd147380fc8924fac4b9d539f85f86f36d4